site/_layouts/doc.liquid

```diff
@@ -286,15 +286,20 @@
                   </ul>
                 </li>
                 <li {% if page.url contains "/account-management/" %} class="open"{% endif %}>
-                  <a href="#" class="sub-menu"><i class="fal fa-fw fa-bullseye"></i> Self Service Account Mgmt<i class="fal fa-chevron-{% if page.url contains "/account-management/" %}up{% else %}down{% endif %} fa-fw"></i></a>
+                  <a href="#" class="sub-menu"><i class="fal fa-fw fa-users"></i> End User UI<i class="fal fa-chevron-{% if page.url contains "/account-management/" %}up{% else %}down{% endif %} fa-fw"></i></a>
                   <ul>
-                    <li {% if page.url == "/docs/v1/tech/account-management/" %}class="active"{% endif %}><a href="/docs/v1/tech/account-management/" >Overview</a></li>
+                    <li {% if page.url contains "/account-management/" %} class="open"{% endif %}>
+                      <a href="#" class="sub-menu">Multi-Factor Auth (MFA)<i class="fal fa-chevron-{% if page.url contains "/account-management/" %}up{% else %}down{% endif %} fa-fw"></i></a>
+                      <ul>
+                        <li {% if page.url == "/docs/v1/tech/account-management/" %}class="active"{% endif %}><a href="/docs/v1/tech/account-management/" >Overview</a></li>
```

Comment:
This page shouldn't contain the link to the 'update user data and password' section, since that is no longer part of that overview.

Reply:
Also the overview page should not mention TOTP (unless that change is part of a different PR).